src/main/java/com/sios/stc/coseng/run/Concurrent.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2015 SIOS Technology Corp. All rights reserved.
+ * This file is part of COSENG (Concurrent Selenium TestNG Runner).
+ * 
+ * COSENG is free software: you can redistribute it and/or modify
```

Comment:
what is the strategy? Sounds like Apache v2 could be a better approach. Anyway what is the strategy?